src/ISeabotConfig.ts

```diff
@@ -0,0 +1,15 @@
+export default interface ISeabotConfig {
+    userVoiceChannels: {
+        groupId: string;
+        triggerChannelId: string;
+    };
+    autoDeleteMessagesInChannels: Array<string> | undefined;
+    telemetryChannels: Array<string> | undefined;
+    channelIds: { [key: string]: string } | undefined;
+    roleIds: {
+        moderator: string;
+        everyone: string;
+        premium: string;
+    };
+    emojiIds: { [key: string]: string } | undefined;
```

Comment:
This config property is also giving me some thought. We have a few commands that rely heavily on custom emotes (`RJ`, `google` / `lmgtfy`, etc) and if this bot ends up in other servers, which it probably won't, we may need to account for that somehow.

For now though, I think it's fine - as long as we maybe add some error handling for emotes that don't exist in the config.

Reply:
I actually abandoned emoji ID lookups already in favor of name lookups. This setting doesn't even appear in the current config file, and should have been removed from the interface.